paimon-web-server/src/test/java/org/apache/paimon/web/server/controller/SysRoleControllerTest.java

```diff
@@ -170,9 +250,102 @@ public void testGetRoleList() throws Exception {
 
         PageR<?> r = ObjectMapperUtils.fromJSON(responseString, PageR.class);
         assertNotNull(r);
+
+        assertTrue(
```

Comment:
Ditto.

Reply:
> Ditto.

Thanks for your review, I have updated this PR